lxd/devlxd.go

```diff
@@ -213,6 +214,10 @@ var devlxdAPIHandler = devLxdHandler{"/1.0", func(d *Daemon, c instance.Instance
 			return response.DevLxdErrorResponse(api.StatusErrorf(http.StatusInternalServerError, err.Error()), c.Type() == instancetype.VM)
 		}
 
+		if state == api.Ready {
```

Comment:
If the application inside the instance later sends `api.Started` as the request state, which will set `volatile.last_state.ready: false` (see above), should we also generate a new event using `lifecycle.InstanceStarted` again?

Reply:
map[string]any{"ready": state == api.Ready}

Reply:
It would be a little counter intuitive, I think, to get an `InstanceReady` event with `ready: false` in it.